### ISSUE TITLE
FYI: cookie syncing

### ISSUE THREAD

**Issue by Atavic** [ghacksuserjs/ghacks-user.js#11]
[TheWebNeverForgets](https://github.com/citp/TheWebNeverForgets) has a paper called [Persistent Tracking Mechanisms in the Wild](https://securehomes.esat.kuleuven.be/%7Egacar/persistent/the_web_never_forgets.pdf) that explains a very underrated issue: **Cookie Syncing**.

**Comment by Roman-Nopantski** [comment]
OK. Now I've finished that pyllyukko vs ghacks comparison, I've had a look. This is old news.

- Canvas: is covered for now by CanvasBlocker (I always fake a unique value, I'm not an expert on the various settings), and the Tor Uplift has some old tickets they might get to in 5 years.
- Flash: Personally, I don't have it at all installed - and that means I'm immune from the respawning via LSO's. For those that do, the user.js section 1800 has plugin.scan.plid.all at false. If they flip that there is still click to play with a zero minute interval (meaning every domain, every flash object is click to play even if you just clicked one) - that should stop some drive-bys. Other than that, hopefully sooner rather than later Flash will be dropped entirely by FF and Chrome - god knows they've been trying.
- Cookies: Personally, I have all cookies blocked by default. I also block all XSS and JS by default. And then I use an addon to allow, allow as 1st party only, allow for session - I use Cookie Controller, because it also controls dom storage. I haven't tried Self Destructing Cookies, but it sounds good as well. We can also have PB windows - ie I start in a normal window, open a PB one, when I close it that PB session ends and everything is destroyed, then open another PB window. Or there is the FPI when it gets sorted and ready for prime time, and we have containers coming as well. And I'm thinking process per tab might do some containment as well, not sure.

tl;dr: Canvas use an addon. Flash, uninstall it. Cookies, block all and use an extension. Cookies are also old school, they use other methods now, because "everyone" knows about cookies and that tired old mantra :)

**Comment by Atavic** [comment]
>This is old news.

Not for the most social users.

**Comment by Gitoffthelawn** [comment]
@Roman-Nopantski 
*"I always fake a unique value,"*

You are using a different unique value for each session, correct?

**Comment by Roman-Nopantski** [comment]
@Gitoffthelawn Yes
